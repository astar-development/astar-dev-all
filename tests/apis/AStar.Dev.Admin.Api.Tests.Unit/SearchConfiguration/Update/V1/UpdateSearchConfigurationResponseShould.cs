using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Admin.Api.SearchConfiguration.Update.V1;

// [TestSubject(typeof(UpdateSearchConfigurationResponse))]
public sealed class UpdateSearchConfigurationResponseShould
{
    [Fact]
    public void ContainTheExpectedValues()
    {
        var sut = new UpdateSearchConfigurationResponse(new());

        sut.ToJson().ShouldMatchApproved();
    }
}
