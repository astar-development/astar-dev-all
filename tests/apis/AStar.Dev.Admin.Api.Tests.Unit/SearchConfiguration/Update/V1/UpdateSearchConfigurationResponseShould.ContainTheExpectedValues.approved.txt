{
  "siteConfigurationSlug": "",
  "searchString": "",
  "subscriptions": "",
  "searchStringPrefix": "",
  "topWallpapers": "",
  "totalPages": 0,
  "imagePauseInSeconds": 0,
  "searchStringSuffix": "",
  "startingPageNumber": 0,
  "useHeadless": false,
  "subscriptionsStartingPageNumber": 0,
  "subscriptionsTotalPages": 0,
  "topWallpapersTotalPages": 0,
  "topWallpapersStartingPageNumber": 0,
  "slowMotionDelayInMilliseconds": 0
}